lib/iex/lib/iex/helpers.ex

```diff
@@ -393,14 +393,19 @@ defmodule IEx.Helpers do
   end
 
   @doc """
-  Prints information about the data type of the given term.
+  Prints information about the data type of any vaild Elixir term.
   
   ## Examples
 
-      i([1,2,3])
-      i("Elixir rocks!")
-      i(:some_atom)
-      i(fn x -> x * 2 end)
+      iex> i(1..5)
+      Term
+        1..5
+      Data type
+        Range
+      Description
+        This is a struct. Structs are maps with a __struct__ key.
+      Reference modules
+        Range, Map
```

Comment:
This can be confusing to read since the format "`iex>` followed by something on the next line" is usually reserved for doctests. Some time ago @michalmuskala made a PR where we discussed this and arrived to the conclusion that a nice way to express this could be something like:

```elixir
@doc """
...

## Examples

    iex> i(1..5)

Will print:

    Term
      1..5
    ...
```

Wdyt?